From: =?utf-8?q?Jonas_Dre=C3=9Fler?= <verdre@v0yd.nl>
Date: Mon, 7 Mar 2022 12:11:06 +0100
Subject: popupMenu: Fix usage of GRAB_NOTIFY ClutterEventFlag

Actually the flag gjs gives us here is called
Clutter.EventFlags.FLAG_GRAB_NOTIFY, so fix that.

Part-of: <https://gitlab.gnome.org/GNOME/gnome-shell/-/merge_requests/2231>

Origin: https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/64e4d5df
---
 js/ui/popupMenu.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/js/ui/popupMenu.js b/js/ui/popupMenu.js
index f9f3a3b..8347103 100644
--- a/js/ui/popupMenu.js
+++ b/js/ui/popupMenu.js
@@ -1374,7 +1374,7 @@ var PopupMenuManager = class {
                 return Clutter.EVENT_STOP;
             }
         } else if (event.type() === Clutter.EventType.ENTER &&
-                   (event.get_flags() & Clutter.EventFlags.GRAB_NOTIFY) === 0) {
+                   (event.get_flags() & Clutter.EventFlags.FLAG_GRAB_NOTIFY) === 0) {
             let hoveredMenu = this._findMenuForSource(targetActor);
 
             if (hoveredMenu && hoveredMenu !== menu)
